packages/flame/lib/src/ai/README.md

```diff
@@ -0,0 +1,189 @@
+
+Most of the code in this directory was ported from [gdxAI] project by libGDX, licensed under
+Apache v2.0. The text of the license is given below.
+
+For now, all ai-related code in Flame should be considered experimental, and its API may change
```

Comment:
This would be an argument to have it in a separate package until it's stable and can be moved in to here? I do see the value of having it directly in here once it's stable.

Reply:
I don't see any advantages to the separation. Clearly, the code is very new and may need to change somewhat in the future, but it doesn't destabilize the rest of the system. So it should be fine, as long as we make it abundantly clear that this part of the code doesn't have the same stability guarantees as the rest the system. I moved the warning into the main documentation page, so that it can't be missed.

In general, we've seen in the past (e.g. `mock_canvas`) that moving things into separate packages, creates a significant burden for development.